middlewares/members.js

```diff
@@ -69,6 +88,33 @@ exports.getUser = async (req, res) => {
     });
 };
 
+exports.getUsersEmail = async (req, res) => {
+    if (!req.permissions.hasPermission('global:mail:member')) {
+        return errors.makeForbiddenError(res, 'Permission global:mail:member is required, but not present.');
+    }
+
+    if (req.query.query && !req.query.query.match(/^\d+(?:,\d+)*$/g)) {
+        return errors.makeBadRequestError(res, 'Query should be a string of 1 id or multiple ids seperated by commas.');
+    }
+
+    let where = {};
+
+    if (typeof req.query.query === 'string' && req.query.query.trim().length > 0) {
+        where = { id: { [Sequelize.Op.or]: req.query.query.split(',') } };
+    }
+
+    const result = await User.findAndCountAll({
+        where,
+        attributes: ['id', 'email', 'gsuite_id', 'primary_email', 'notification_email']
+    });
+
+    return res.json({
+        success: true,
+        data: result.rows, // TODO: only return id and notification_email
```

Comment:
@serge1peshcoff any ideas on this? Removing email/gsuite_id/primary_email from attributes above does not work since it can not calculate the virtual notification_email anymore